### ISSUE TITLE
@Daniel 深度碎片 第一课笔记汇总

### ISSUE THREAD

**Issue by EmbraceLife** [EmbraceLife/shendusuipian#63]
1. [Lesson 0 How to get GPU running](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=Lesson%200%20How%20to%20get%20GPU%20running)
2. 如何启动你的第一个GPU
3. [What else do you need to get started?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20else%20do%20you%20need%20to%20get%20started%3F)
4. 你需要做些什么准备？
5. [How to make the most out of these lesson videos and notebooks?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20make%20the%20most%20out%20of%20these%20lesson%20videos%20and%20notebooks%3F)
6. 如何最大化利用课程视频与notebook?
7. [what will you become and key resources?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=what%20will%20you%20be%20and%20key%20resources%3F)
8. 你可以期望的学习成就高度与关键学习资源
9. [why should we learn from Jeremy Howard?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=why%20should%20we%20learn%20from%20Jeremy%20Howard%3F)
10. 为什么我们要跟着Jeremy Howard学习？
11. [How to make DL accessible to everyone to do useful things?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20make%20DL%20accessible%20to%20everyone%20to%20do%20useful%20things%3F)
12. fast.ai让人人都成为深度学习高手的策略方针
13. [How much to invest and What I get out?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20much%20to%20invest%20and%20What%20I%20get%20out%3F)
14. 需要投入多少精力，会有怎样的收获？
15. [Prerequisites and False assumptions and claims on DL](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=Prerequisites%20and%20False%20assumptions%20and%20claims%20on%20DL)
16. 学习深度学习的必要基础是什么，以及常见的误解与成见
17. [What will you be able to do at the end of lesson 1](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20will%20you%20be%20able%20to%20do%20at%20the%20end%20of%20lesson%201)
18. 完成第一课后你将能做些什么
19. [What is fastai learning philosophy](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20is%20fastai%20learning%20philosophy)
20. fast.ai的教学哲学是什么
21. [How to use Jupyter notebook as a pro? ](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20use%20Jupyter%20notebook%20as%20a%20pro%3F)
22. 如何像专业人士一样使用jupyter notebook
23. [What are Jupyter magics?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20are%20Jupyter%20magics%3F)
24. 什么是Jupyter magics
25. [What are fastai lib and how to use it?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20are%20fastai%20lib%20and%20how%20to%20use%20it%3F)
26. 什么是fastai库以及如何使用
27. [Academic vs Kaggle Datasets, CatsDogs vs Pets dataset](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=Academic%20vs%20Kaggle%20Datasets,%20CatsDogs%20vs%20Pets%20dataset)
28. 学术和Kaggle数据，CatsDogs 与 Pets数据的异同
29. [How to download dataset with fastai](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20download%20dataset%20with%20fastai)
30. 如何下载数据包
31. [How to access image folders and check filenames inside](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20access%20image%20folders%20and%20check%20filenames%20inside)
32. 如何进入图片文件夹以及查看里面的文件
33. [How to get the labels of dataset?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20get%20the%20labels%20of%20dataset%3F)
34. 如何从文件名中获取标注
35. [Why and how to pick the image size for DataBunch?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=Why%20and%20how%20to%20pick%20the%20image%20size%20for%20DataBunch%3F)
36. 为什么以及如何选择图片大小规格
37. [What is a DataBunch](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20is%20a%20DataBunch)
38. 什么是DataBunch
39. [What does normalize do to `DataBunch`?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20does%20normalize%20do%20to%20%60DataBunch%60%3F)
40. 如何normalize DataBunch
41. [What to do if `size` is not 224?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20to%20do%20if%20%60size%60%20is%20not%20224%3F)
42. 如果图片大小不是224会怎样
43. [What does it mean to normalize images](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20does%20it%20mean%20to%20normalize%20images)
44. normalize 图片意味着什么
45. [Why 224 not 256 as power of 2 to be image size](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=Why%20224%20not%20256%20as%20power%20of%202%20to%20be%20image%20size)
46. 为什么图片尺寸是224而不是256
47. [How to check the real images and labels](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20check%20the%20real%20images%20and%20labels)
48. 如何查看图片和标注
49. [How to build a CNN learner/model?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20build%20a%20CNN%20learner/model%3F)
50. 如何构建一个CNN模型
51. [Why use a pretrained model (framework and parameters) for your CNN? in other words, What is transfer learning?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=Why%20use%20a%20pretrained%20model%20%28framework%20and%20parameters%29%20for%20your%20CNN%3F%20in%20other%20words,%20What%20is%20transfer%20learning%3F)
52. 为什么要用训练好的模型的框架与参数
53. [what is overfitting? why wouldn’t the model cheating?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=what%20is%20overfitting%3F%20why%20wouldn%E2%80%99t%20the%20model%20cheating%3F)
54. 什么是过拟合？为什么我们的模型很难过拟合？
55. [How to train the model with the best technique?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20train%20the%20model%20with%20the%20best%20technique%3F)
56. 如何用最优的技术来训练模型？
57. [How to find out how good is your model?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20find%20out%20how%20good%20is%20your%20model%3F)
58. 如何了解模型的好坏？
59. [How to get the most out of this course?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20get%20the%20most%20out%20of%20this%20course%3F)
60. 如何放大课程效果？
61. [The popularity of fastai library and replacing Keras](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=The%20popularity%20of%20fastai%20library%20and%20replacing%20Keras)
62. fastai的业界口碑以及与kera的对比
63. [What students achieved with fastai and this course?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=What%20students%20achieved%20with%20fastai%20and%20this%20course%3F)
64. 学员能够用fast.ai所学完成的项目
65. [Why use Resnet rather than Inception?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=Why%20use%20Resnet%20rather%20than%20Inception%3F)
66. 为什么选择ResNet 而非Inception作为模型框架和已训练的参数
67. [How to save a trained model?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20save%20a%20trained%20model%3F)
68. 如何保存训练好的模型
69. [how to plot top losses examples/images?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=how%20to%20plot%20top%20losses%20examples/images%3F)
70. 如何画出损失值最高的数据/图？
71. [How to find out the most confused images of our model?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20find%20out%20the%20most%20confused%20images%20of%20our%20model%3F)
72. 如何找出模型混淆度最高的图片？
73. [How to improve our model? finetuning](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20improve%20our%20model%3F%20finetuning)
74. 如何用微调改进模型？
75. [what is CNN actually learning and why previous full model training didn’t work](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=what%20is%20CNN%20actually%20learning%20and%20why%20previous%20full%20model%20training%20didn%E2%80%99t%20work)
76. CNN模型在学习些什么？以及为什么直接用全模型学习效果不佳？
77. [How to train the whole model in the right way?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20train%20the%20whole%20model%20in%20the%20right%20way%3F)
78. 训练全模型的正确方式是什么？
79. [How to improve model with more layers?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20improve%20model%20with%20more%20layers%3F)
80. 如何用更大的模型来改进效果
81. [Different ways to put your data into DataBunch](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=Different%20ways%20to%20put%20your%20data%20into%20DataBunch)
82. 生成DataBunch的不同方式有哪些
83. [How to make the most out of documents?](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=How%20to%20make%20the%20most%20out%20of%20documents%3F)
84. 如何最大化利用fastai文档
85. [QA on fastai with multi-GPU, 3D data](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=QA%20on%20fastai%20with%20multi-GPU,%203D%20data)
86. 关于fastai与多GPU和3D数据的问题
87. [An interesting and inspiring project](bear://x-callback-url/open-note?id=F689DB5F-22DE-4675-90CB-5E570C254441-11295-0001BFBDBAD7426A&header=An%20interesting%20and%20inspiring%20project)
88. 一些有趣的项目
89.
